tests/integration/olp-cpp-sdk-authentication/HereAccountOauth2Test.cpp

```diff
@@ -23,13 +23,15 @@
 #include <matchers/NetworkUrlMatchers.h>
 #include <mocks/NetworkMock.h>
 #include <olp/authentication/AuthenticationClient.h>
-#include <olp/authentication/AutoRefreshingToken.h>
+#include <olp/authentication/TokenProvider.h>
 #include <olp/core/client/OlpClientSettingsFactory.h>
 #include <olp/core/http/HttpStatusCode.h>
 #include <olp/core/porting/make_unique.h>
 #include <olp/core/porting/warning_disable.h>
 
```

Comment:
Remove

Reply:
I moved these constants to TokenProvider:
static constexpr auto kDefaultMinimumValidity = 300ll;
static constexpr auto kDefaultMinimumValiditySeconds =
    std::chrono::seconds(kDefaultMinimumValidity);
static constexpr auto kForceRefresh = std::chrono::seconds(0);